blueocean-events/src/main/java/io/jenkins/blueocean/events/PipelineEventListener.java

```diff
@@ -172,6 +196,22 @@ private void publishEvent(Message message) {
         }
     }
 
+    @Extension
+    public static class InputStepPublisher implements PipelineInputStepListener {
+
+        @Override
+        public void onStepContinue(InputStep inputStep, WorkflowRun run) {
+            // fire an unpaused event in case the input step has received its input
+            try {
+                PubsubBus.getBus().publish(new RunMessage(run)
+                        .setEventName(Events.JobChannel.job_run_unpaused)
```

Comment:
this is just to prompt the front end to go fetch its new state, not the actual final word on it, so this should be ok. Its just a notification really (like "you've got mail").